source/reference/access-mdb-shell-help.txt

```diff
@@ -0,0 +1,241 @@
+Access the |mdb-shell| Help
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+``mongosh`` provides additional help information in its "online"
+help system. This document provides an overview of accessing this help
+information. 
+
+.. _mdb-shell-help-command-line:
+
+Command Line Help
+-----------------
+
+To see the list of options and help for starting ``mongosh``, use the
+``--help`` option from the command line:
+
+.. code-block:: sh
+
+   mongosh --help
+
+Shell Help
+----------
+
+To see the list of help, in ``mongosh``, type ``help``:
+
+.. code-block:: javascript
+
+   help
+
+.. _mdb-shell-help-db:
+
+Database Help
+-------------
+
+In ``monogsh``:
+
+- To see the list of databases on the server, use the ``show dbs``
+  command:
+
+  .. code-block:: javascript
+
+     show dbs
+
+  ``show databases`` is an alias for ``show dbs``.
+
+- To see the list of help for methods you can use on the ``db``
+  object, call the ``db.help()`` method or run the ``db.help`` command:
+
+  .. include:: /includes/admonitions/help-interchangeable.rst
+
+  .. code-block:: javascript
+
+     db.help()
+
+  The output resembles the following:
+
+  .. code-block:: none
+     :copyable: false
+
+     Database Class:
+
+       getMongo                Returns the current database connection
+       getCollectionNames      Returns an array containing the names of
+                               all collections in the current database.
+       getCollectionInfos      Returns an array of documents with
+                               collection information, i.e. collection
+                               name and options, for the current
+                               database.
+       runCommand              Runs an arbitrary command on the
+                               database. 
+       adminCommand            Runs an arbitrary command against the
+                               admin database. 
+       aggregate               Runs a specified admin/diagnostic
+                               pipeline which does not require an
+                               underlying collection.
+       getSiblingDB            Returns another database without
+                               modifying the db variable in the shell
+                               environment. 
+       getCollection           Removes the current database, deleting
+                               the associated data files.
+       dropDatabase            Removes the current database, deleting
+                               the associated data files.
+
+- To see help for a specific database method in ``mongosh``, type the
+  ``db.<method name>`` without the parenthesis (``()``), followed by
+  ``.help`` or ``.help()``. The following example returns help for the
+  :method:`db.adminCommand()` method:
+
+  .. include:: /includes/admonitions/help-interchangeable.rst
```

Comment:
What do you think about rather than having this tip spread out throughout the page, we just have a top-level blurb with something like:

"When accessing help information in ``mongosh``, ``.help`` and ``.help()`` syntaxes are interchangeable"

I understand the benefit of helping folks who jump to a specific section, but this info isn't _super_ necessary (just nice to know) so I think just calling it out once at the top of the page is fine.

Reply:
I asked this question myself. Had decided that it would be harder to miss if it were everywhere. But you're right -- it's overkill.

Updated.